cloud-common/src/main/java/com/sequenceiq/cloudbreak/common/service/DefaultCostTaggingService.java

```diff
@@ -88,4 +88,10 @@ private String transform(String value, String platform) {
         return result;
     }
 
+    private DateTimeFormatter getUtcFormatter(){
```

Comment:
This method isn't relates to the class, and also a reusable part, so would be better in a separated helper/util/service.

Reply:
I agree :). I just was not sure how much people here like very small classes => I move it to a distinct class.